src/camera_session.cpp

```diff
@@ -76,7 +89,7 @@ int CameraSession::init() {
 
   config_ = std::move(cfg);
 
-  return 0;
+  return true;
```

Comment:
We return true, 2 and 0 here. We should either go all int or all bool. 0 for success and two other numbers for the two other cases would be the most standard way.